clib/src/main/scala/scala/scalanative/libc/stdint.scala

```diff
@@ -4,10 +4,26 @@ import scala.scalanative.unsafe._
 
 @extern object stdint extends stdint
 
+// A very partial implementation. May it grow to completeness with time.
+
 @extern private[scalanative] trait stdint {
   // intmax_t and uintmax_t are not always equivalent to `long long`,
   // but they are usually `long long` in common data models.
   type intmax_t = CLongLong
   type uintmax_t = CUnsignedLongLong
 
+  /* Assume & implement a data model where the size of a C/C++ pointer is the
+   * same as the maximum size of a C/C++ data object.
+   * 
+   * C and C++ do not __require__ this match but it is empirically true
+   * for the most common data models. This is similar to the intmax_t/uintmax_t
+   * assumption above.
+   *
+   * Reference:
+   *   https://en.wikipedia.org/wiki/64-bit_computing#64-bit_data_models
+   *   Retrieved 2025-10-31
+   */
+
+  type intptr_t = CSSize
```

Comment:
i found that `type atomic_intptr_t = CSSize` so this was done before :)

Reply:
Good find.  The seal having been broken, we can do it a second time?

This time around there will/should be an acknowledgement of the abstraction
break so that future devos can easily know such has happened and they are 
in 'unsafe' territory and can stop tracing without having to figure the abstraction
break all over again on their own.